### PR TITLE
[FEATURE] allow fallback upload folder for all tables

### PR DESCRIPTION
**Pull request by franzkugelmann**

### CODE REVIEW

File: Configuration/PageTs/defaultUploadFolders.ts [hunk 1]
```diff
@@ -8,4 +8,7 @@ default_upload_folders {
     # You can set a folder for the whole table of for a specific field of that table
     tx_news_domain_model_news.fal_related_files = news_downloads
     tx_news_domain_model_news.fal_media = news_media
+
+    # You can set a fallback for all tables
+    DefaultForAllTables = 1:myDefaultUploadFolderForThisPartOfTheTree
```
Comment (frans-beech-it) [inline Configuration/PageTs/defaultUploadFolders.ts]: Can you change this so it starts lower case like all other settings: `defaultForAllTables`.

**Comment by frans-beech-it** [comment]
tnx for the improvement. could you make the little adjustment in the settings name? After that I will merge it into the project

**Comment by franzkugelmann** [comment]
Yepp :) Already wondered if i should use that with lowerCase.

**Comment by frans-beech-it** [comment]
tnx for this improvement

**Pull request merged**